docs/dev-tools/intro-webhooks.md

```
@@ -6,135 +6,421 @@ keywords: plane, developer tools, integrations, extensions, webhooks, automation
+| Work item type | `type_id`      | Work item type UUID                                             |
+| State group    | `state_group`  | `backlog` · `unstarted` · `started` · `completed` · `cancelled` |
+| Assignees      | `assignee_id`  | User UUID                                                       |
+| Labels         | `label_id`     | Label UUID                                                      |
+| Projects       | `project_id`   | Project UUID                                                    |
+| Priority       | `priority`     | `none` · `low` · `medium` · `high` · `urgent`                   |
+| Start date     | `start_date`   | ISO date                                                        |
+| Due date       | `target_date`  | ISO date                                                        |
+
+**Expression syntax**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)